README.md

```diff
@@ -80,6 +80,7 @@ Visit [Eiffel Community](https://eiffel-community.github.io) to get started and
    1. [Build Avoidance](./usage-examples/build-avoidance.md)
    1. [Pipeline Monitoring](./usage-examples/pipeline-monitoring.md)
    1. [Test Execution](./usage-examples/test-execution.md)
+   1. [Describe Confidence for SW and HW](./usage-examples/clm-for-hw-sw.md)
```

Comment:
The heading doesn't match the heading of the linked page. All other examples in this group keep the name of the heading here in their linked pages, amending "Example" to it. Also, headings should be in upper case, and I believe we should not use abbreviations like HW and SW here.
```suggestion
   1. [Confidence Level for Software and Hardware Combination](./usage-examples/clm-for-hw-sw.md)
```